Xabe.FFmpeg/Probe/IProbe.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Threading.Tasks;
+
+namespace Xabe.FFmpeg
+{
+    /// <summary>
+    ///     Allows to prepare and start IProbe.
+    /// </summary>
+    public interface IProbe
```

Comment:
change this to FFprobe

Reply:
Class for conversion is not FFConversion so I think Probe is better name